.github/ISSUE_TEMPLATE/release-status.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Release Status
+about: Issue template for the status document that we direct our customers to during a release cycle
+title: <month> <year> Release Status per Platform, Version & Binary Type
+labels: ''
+assignees: ''
+
+---
+
+Sharing information in this issue since the TCK work is being tracked in temurin-compliance private repo not visible to the community (as per the OCTLA).  Risks and expectations for timing on the release are listed in https://github.com/adoptium/adoptium/issues/3#issuecomment-866903922.  Primary platforms (x64 Linux/Windows/OSX) in **bold** are prioritized, secondary platforms not in bold follow in no particular order (as machine resources are available).
```

Comment:
```suggestion
Sharing information in this issue since the TCK work is being tracked in temurin-compliance private repo not visible to the community (as per the OCTLA).  Risks and expectations for timing on the release are listed in this [issue comment](https://github.com/adoptium/adoptium/issues/3#issuecomment-866903922).  Primary platforms (x64 Linux/Windows/OSX) in **bold** are prioritized, secondary platforms not in bold follow in no particular order (as machine resources are available).
```